cmd/neofs-cli/modules/netmap/netinfo.go

```diff
@@ -55,7 +54,7 @@ var netInfoCmd = &cobra.Command{
 
 		cmd.Println("NeoFS network configuration (other)")
 		netInfo.IterateRawNetworkParameters(func(name string, value []byte) {
-			cmd.Printf(format, name, hex.EncodeToString(value))
+			cmd.Printf(format, name, string(value))
```

Comment:
The old code was correct here. We don't know if it's a string at all, it can be numeric. This just needs to be explained.

Additionally we can check for UTF-8 and print raw string as well if so, just to make things more human-friendly.

Reply:
do not agree that was a bug too. any system settings that are going to be read by the NeoFS system (and neofs-cli is a NeoFS tool so it should work according to its rules) are printed above so this line is just the best effort: we do not know exactly what it is but we can print _these bytes_ (they are for some NeoFS extensions more likely)

OR

we may add a restriction that the [values](https://github.com/nspcc-dev/neofs-api/blob/7ae1a7438b19fc4ca32c451db67de170edb539a7/netmap/types.proto#L306) are utf8 strings too

Reply:
but +1 that we can check if it is a valid utf-8 string and print it

Reply:
They're explicitly not utf8 in case of numeric settings (like fees). Eventually we'll solve nspcc-dev/neofs-contract#314 and all of this will fade away, but currently in this particular case we can expect arbitrary bytes.

Reply:
> They're explicitly not utf8 in case of numeric settings (like fees).

oh, well, i see. also, changing API isnt a good think in general. so i would document `hex` encoding in CLI